paasta_tools/utils.py

```diff
@@ -81,6 +81,8 @@
 
 INSTANCE_TYPES = ('marathon', 'chronos', 'paasta_native', 'adhoc')
 
+DOCKER_ARGS_WHITELIST = ('shm-size')
```

Comment:
The intent here was to protect against adding --privileged (or similar).  But if we're going to add --privileged anyways, there's not much purpose to this.